templates/tutorials/tutorial.html

```diff
@@ -0,0 +1,224 @@
+          </div>
+          {% endif %}
+
+          <hr class="u-sv3">
+          <footer class="p-tutorial-section__footer row u-no-padding--left u-no-padding--right">
+            <div class="col-6 col-small-2 col-medium-3 u-vertically-center">
+              <a class="p-tutorial__bug-link" href="https://discourse.jujucharms.com{{ document.topic_path }}">
+                <small>Suggest changes&nbsp;&rsaquo;</small>
+              </a>
+            </div>
+            <div class="col-6 col-small-2 col-medium-3 u-align--right">
+              <div class="p-tutorial__duration">
+                <small>
+                  <span class="u-hide--small">about</span>
+                  {% if section["remaining_duration"] %}
+                    {{ section["remaining_duration"] }}
+                  {% else %}
+                    0
+                  {% endif %}
+                  minutes to go
+                </small>
+              </div>
+              <div class="p-tutorial__pagination">
+                {% if loop.first %}
+                  <button class="p-tutorial__pagination-item--prev p-button--neutral has-icon u-no-margin--bottom" disabled style="margin-right: 1rem;">
+                    <i class="p-icon--contextual-menu">Previous step</i>
+                  </button>
+                {% else %}
+                  <a href="#{{ loop.index - 1 }}-{{ loop.previtem['slug'] }}" class="p-tutorial__pagination-item--prev p-button--neutral has-icon u-no-margin--bottom" style="margin-right: 1rem;">
+                    <i class="p-icon--contextual-menu">Previous step</i>
+                  </a>
+                {% endif %}
+
+                {% if loop.last %}
+                  <button class="p-tutorial__pagination-item--next p-button--neutral has-icon u-no-margin--bottom" disabled>
+                    <i class="p-icon--contextual-menu">Next step</i>
+                  </button>
+                {% else %}
+                  <a href="#{{ loop.index + 1 }}-{{ loop.nextitem['slug'] }}" class="p-tutorial__pagination-item--next p-button--neutral has-icon u-no-margin--bottom">
+                    <i class="p-icon--contextual-menu">Next step</i>
+                  </a>
+                {% endif %}
+              </div>
+            </div>
+          </footer>
+        </section>
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- end of tutorial feedback -->
+<script>
+  (function() {
+    var tutorialFeedbackOptions = document.querySelector('.p-tutorial__feedback-options');
+    var tutorialFeedbackIcons = document.querySelectorAll('.p-tutorial__feedback-icon');
+    var tutorialFeedbackResult = document.querySelector('.p-tutorial__feedback-result');
+
+    [].forEach.call(tutorialFeedbackIcons, function(icon) {
+      icon.addEventListener('click', function(e) {
+        var feedbackValue = e.target.getAttribute('data-feedback-value');
+        dataLayer.push({
```

Comment:
Because `dataLayer` isn't defined in this script tag you can't rely on it being here. Can you wrap this in a conditional plz.